src/html/utils.ml

```diff
@@ -10,3 +10,25 @@ let rec list_concat_map ?sep ~f = function
     match sep with
     | None -> hd @ tl
     | Some sep -> hd @ sep :: tl
+
+(**
```

Comment:
I left this code in here, since there were some discussions on how to introduce `refmt` without coupling it to the odoc core